jplag.endToEndTesting/src/main/java/de/jplag/endToEndTesting/constants/Constant.java

```diff
@@ -0,0 +1,25 @@
+package de.jplag.endToEndTesting.constants;
+
+import java.nio.file.Path;
+
+public final class Constant {
+
+	private Constant() {
+		// No need to instantiate the class, we can hide its constructor
```

Comment:
For consistency, use `// private constructor to prevent instantiation`

Reply:
Done